test/views/html/pages/webchat/AdditionalNeedsHelpViewSpec.scala

```diff
@@ -31,6 +32,7 @@ class AdditionalNeedsHelpViewSpec extends ChatViewBehaviours {
 
     behave like normalPage(
       createView,
+      "Ask HMRC - Webchat",
```

Comment:
These should still be the message key ("global.nav.title"), since it's passed through messages in normalPage. It happens to work as it doesn't find a key for the literal string you're passing, but that seems a bit tacky.